lib/improver/orographic_enhancement.py

```diff
@@ -470,9 +470,9 @@ def _add_upstream_component(self, point_orogenh):
 
         return orogenh
 
-    def _create_output_cubes(self, orogenh_data, reference_cube):
+    def _create_output_cube(self, orogenh_data, reference_cube):
         """
-        Create two output cubes of orographic enhancement on different grids.
+        Creates a cube of orographic enhancement on a grid.
         Casts coordinate points and bounds explicitly to np.float32.
```

Comment:
Can probably cut this to `Creates a cube containing orographic enhancement values in SI units`.

Reply:
Done